microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,114 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+
+
+<!-- ------------------------------ -->
+#### Sensitive information that's not sent
+
+<!-- The Privacy settings have been updated from what is described here (assume comment means h2 section) -->
+
+If Microsoft Edge detects typing in the Address bar that might contain sensitive information, Microsoft Edge doesn't send the typed text.  Sensitive information includes:
+* Authentication credentials.
+* Local file names.
+* URL data that's normally encrypted.
+
+
+<!-- ------------------------------ -->
+#### Enabling collecting Address bar diagnostic data
+
+You can configure Microsoft Edge to collect diagnostic data about the Address bar.  Collected data includes the number of offered queries for all search providers.  To enable collecting diagnostic data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, and then in **Personalize your web experience**, turn on the setting **Improve your web experience by allowing Microsoft to use your browsing history from this account for personalizing advertising, search, news, and other Microsoft services**.
+
 
-You can configure Microsoft Edge to collect diagnostic data about the address bar. Collected data includes the number of offered queries for all search providers. Go to `edge://settings/privacy`. In **Personalize your web experience** turn on the **Improve your web experience by allowing Microsoft to use your browsing history from this account for personalizing advertising, search, news, and other Microsoft services** setting.
+<!-- ------------------------------ -->
+#### Deleting data
+
+Typed characters and the websites that you visit are stored locally on the device, per profile.  To delete that data, go to `edge://settings/clearBrowserData`, and then in the **Clear browsing data** window, select the **Browsing history** checkbox.  Then click the **Clear now** button.
 
-Typed characters and the websites you visit are stored locally on the device per profile.  To delete the data, go to `edge://settings/clearBrowserData`. In the **Clear browsing data** window, select the **Browsing history** checkbox, and select the **Clear now** button.
+If Bing is your default search provider and you are signed in, you can delete your searches by using the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  You can clear your browsing history and delete websites from appearing as suggestions in the Address bar: go to `edge://history`, and then select **Clear browsing data**.  You can delete the data that Microsoft collects from the Address bar and search suggestions features on Windows 10 and later: open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**, and then in **Delete diagnostic data**, select **Delete**.<!-- deleting diagnostic data could use rewrite -->  All other data is deleted after 36 months.
```

Comment:
The steps to delete browsing history should be the same is this paragraph and the one above it.

Reply:
snapshot of result at the moment: 
i added comment asking if paragraph can be reformatted as list.  that design decision is affected by the in-question "Start" menu sentence.
```
<!-- ------------------------------ -->
#### Deleting data

Typed characters and the websites that you visit are stored 
locally on the device, per profile.  To delete that data, 
select **Settings and more** (![Settings and more 
icon](./index-images/settings-and-more-icon.png)) > 
**Settings** > **Privacy, search, and services**, and then 
in the **Delete browsing data** section, click the **Choose 
what to clear** button.  The **Delete browsing data** dialog 
opens.  Click the **Clear now** button.

If Bing is your default search provider and you are signed in,
<!-- todo: reformat paragraph as 3 list items starting here? --> 
you can delete your searches by using the [Microsoft Privacy 
dashboard](https://account.microsoft.com/privacy/).  To clear 
your browsing history and delete websites from appearing as 
suggestions in the Address bar: select **Settings and more** 
(![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy, search, and services**, and then in 
the **Delete browsing data** section, click the **Choose what 
to clear** button.  The **Delete browsing data** dialog 
opens.  Click the **Clear now** button..  You can delete the data 
that Microsoft collects from the Address bar and search 
suggestions features on Windows 10 and later: open **Start** > 
**Settings** > **Privacy** > **Diagnostics & feedback**, and then 
in **Delete diagnostic data**, select **Delete**.<!-- todo: is this 
sentence about Start menu for the correct feature?  seems to be 
about deleting Edge data from within the OS rather than from 
within Edge --><!-- old comment: deleting diagnostic data could 
use rewrite -->  All other data is deleted after 36 months.<!-- todo: 
what is "all other data"? -->
```

Reply:
in both paragraphs, i'll try merging the "dialog opens" step inline forming 1 sentence...

Reply:
result 2x:
```
, select **Settings and more** (![Settings and more icon]
(./index-images/settings-and-more-icon.png)) > 
**Settings** > **Privacy, search, and services**, and then 
in the **Delete browsing data** section, click the **Choose 
what to clear** button; then in the **Delete browsing data** 
dialog, click the **Clear now** button.
```